### ISSUE TITLE
manual/scalaGuide/main/json/ScalaJsonTransformers.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#1018]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/main/json/ScalaJsonTransformers.md
- diffs between [2.2.0/manual/scalaGuide/main/json/ScalaJsonTransformers.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/scalaGuide/main/json/ScalaJsonTransformers.md) and [2.3.5/manual/scalaGuide/main/json/ScalaJsonTransformers.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/main/json/ScalaJsonTransformers.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/scalaGuide/main/json/ScalaJsonTransformers.md   2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/scalaGuide/main/json/ScalaJsonTransformers.md  2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # <a name="json-to-json">JSON transformers</a>

 > Please note this documentation was initially published as an article by Pascal Voitot ([@mandubian](https://github.com/mandubian)) on [mandubian.com](http://mandubian.com/2012/10/29/unveiling-play-2-dot-1-json-api-part3-json-transformers/)
@@ -368,6 +369,7 @@

 '''
 import play.api.libs.json._
+import play.api.libs.json.Reads._

 val jsonTransformer = (__ \ 'key2).json.pickBranch(
   (__ \ 'key21).json.update( 
@@ -510,6 +512,7 @@

 '''
 import play.api.libs.json._
+import play.api.libs.json.Reads._
 import play.api.libs.functional.syntax._

 val gizmo2gremlin = (
@@ -550,4 +553,4 @@
 - It merges results of both `Reads[JsObject]` (JsObject ++ JsObject)
 - It also applies the same JSON to both `Reads[JsObject]` unlike `andThen` which injects the result of the first reads into second one.

-> **Next:** [[JSON Macro Inception | ScalaJsonInception]]
\ No newline at end of file
+> **Next:** [[JSON Macro Inception | ScalaJsonInception]]

```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1111
